tests/test_cli.py

```diff
@@ -65,30 +75,28 @@ def runner():
 def test_success_download(runner, mock_get_datasets, mock_requests_file_get):
     print(f"Try the get_datasets() mock: {machine_data_hub.cli.get_datasets('hello')}")
     print(f"Try the request.get mock: {requests.get('hello').json()}")
-    result = runner.invoke(app, ["download", "Combined Cycle Power Plant Data Set"])
+    result = runner.invoke(app, ["download", 1, 1])
```

Comment:
the way this invoke command works, it expects a list of strings even when you pass numbers.

The tests are passing locally when I change to `["download", "1", "1"]`